joomla/joomla-cms
# Add a Joomla! Submit Testing Results Extension

## Situation today
While we are in Alpha, Beta, RC development mode we don’t have a   real overview how the packages are testet. We get some results when people submit issues but we don’t have an idea if people are testing in real live environments or if people are testing at all 

One of the problems is that there is no place someone can report a successful test result. This PR should solve this problem.

## Idea

The idea is to have a specific extension for reporting test results. The date will be sent to the statistics server and we don’t ask people for personal data (they can submit personal data, I will explain the idea behind this later)

It is as easy as possible, people ask simple question like „Is this a copy of a production site or a vanilla test site“, „Does the site run in a production like environment“, „Did you update or is it a clean install“ and so on. Just some easy questions. The next step is to report if someone tested a part of the CMS Yes with or without success or no.

The testing result allow us to have a better understanding, what people are testing and where the package has problems. This will give us the chance to work on specific areas and improve the quality. 

The extension will only be enabled when we are in development mode (alpha, beta, rc) it doesn’t makes sense to have the extension enabled for a stable version of the CMS.

I added some fields for personal data but these are not required, but if people give us there name we can list them as „PreReleaseTesters“ or if they give us the github account we can add points for testing so that people get some „fame“ back.

# How to test

* Apply patch
* Make a discover install
* play around

If you hit send then you will see what would be send to the server. The endpoint isn’t developed at the moment, I though I will look first if people like the idea.
